src/hound/vcs/svn.go

```diff
@@ -0,0 +1,79 @@
+package vcs
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func init() {
+	RegisterVCS("svn", &SVNDriver{})
+}
+
+type SVNDriver struct{}
+
+func (g *SVNDriver) HeadHash(dir string) (string, error) {
+	cmd := exec.Command(
```

Comment:
I'm in process of fixing something right now that will make it possible to use the output of svnversion. Currently, Hound expects versions to be 40 character hashes. I think svnversion returns svn's serial revision number.